django/contrib/postgres/fields/ranges.py

```diff
@@ -199,23 +199,32 @@ def as_sql(self, compiler, connection):
 class RangeContainedBy(lookups.PostgresSimpleLookup):
     lookup_name = 'contained_by'
     type_mapping = {
+        'smallserial': 'int4range',
+        'serial': 'int4range',
+        'bigserial': 'int8range',
```

Comment:
Ahh just realized that `AutoFieldMixin.output_field` doesn't return it's implementing `IntegerField` subclasses so it's likely not fixed in 3.0 without these lines like I wrongly assumed.